src/interface.jl

```diff
@@ -524,3 +525,13 @@ function postsolve(tree, result, time_ref, verbose=false)
 
     return x
 end
+
+function free_model(o::SCIP.Optimizer)
+    SCIP.SCIPfreeTransform(o)
+end
+
+function free_model(o::MOI.ModelLike)
+    if (!(MOI.is_empty(o)))
+        MOI.empty!(o)
```

Comment:
one gotcha here, FreeTransform just removes an inner level of cache in SCIP, but the outer model is still there, otherwise normally the model should not be erased, I'm surprised this does not cause an error

Reply:
The error is avoided by moving the command at the end of `postsolve()` function. 

i) Is it fine to leave it there or should I move it back where it was (i.e., line 453 in interface.jl)?
ii) In general, should we try to do something similar to HiGHS model as the current code does for the SCIP model `o`, or we should leave the `o` model untouched in the case of HiGHS?

Reply:
I think ii